conf/application.conf

```diff
@@ -87,11 +87,16 @@ features {
   hideNavbarLogin = false
   # if isDemoInstance is true, / will always be the /spotlight page
   # if it is false, / will be the login page, if not logged in, and the dashboard otherwise
-  isDemoInstance = false
+  isDemoInstance = true
   autoBrushReadyDatasets = []
   taskReopenAllowedInSeconds = 30
   allowDeleteDatasets = true
   jobsEnabled = false
+  # For new users, the dashboard will show a banner which encourages the user to check out the following dataset.
+  # If isDemoInstance == true, `/createExplorative/hybrid/true` is appended to the URL so that a new tracing is opened.
+  # If isDemoInstance == false, `/view` is appended to the URL so that it's opened in view mode (since the user might not
+  # have an account).
+  publicDemoDatasetUrl = "https://webknossos.org/datasets/MPI_Brain_Research/2012-09-28_ex145_07x2_ROI2017_connectome"
```

Comment:
The internal feedback was to use this dataset instead: https://webknossos.org/datasets/scalable_minds/l4dense_motta_et_al_demo

Reply:
Good catch!